### PR TITLE
[Procuration] Show disable mode of procuration request

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: templates/procuration_manager/requests.html.twig [hunk 1]
```diff
@@ -41,6 +46,9 @@
                     <th class="datagrid__table__col--left">Scrutins concernés</th>
                     <th>Mandataires</th>
                     <th class="datagrid__table__col--hide-mobile">Date de la demande</th>
+                    {% if constant('DISABLED', filters) == filters.status %}
+                        <th>Désactivée</th>
+                    {% endif %}
```
Comment (elvino29) [inline templates/procuration_manager/requests.html.twig:51]:
```suggestion
                      <th>Désactivée</th>
```

File: tests/Controller/EnMarche/ProcurationManagerControllerTest.php [hunk 2]
```diff
@@ -283,6 +283,7 @@ public function testProcurationManagerRequestsListDisabled()
         $crawler = $this->client->request(Request::METHOD_GET, '/espace-responsable-procuration?status=disabled');
 
         $this->isSuccessful($this->client->getResponse());
+        $this->assertStringContainsString('Néanmoins certains mandants ont été désactivés de manière automatique pour des raisons de sécurité, merci de ne pas les réactiver.', trim($crawler->filter('.procuration-manager div.alert--tips')->text()));
```
[the remark this answers is not in the document]
Comment (illusionOfParadise) [reply tests/Controller/EnMarche/ProcurationManagerControllerTest.php:286]: The text has been provided by PO.

File: templates/procuration_manager/_requests_list.html.twig [hunk 3]
```diff
@@ -34,6 +34,11 @@
         <td class="datagrid__table__col--hide-mobile">
             {{ request.data.createdAt|date('d/m/Y H:i') }}
         </td>
+        {% if constant('DISABLED', filters) == filters.status %}
+            <td>
+                {{ request.data.disabledReason == constant('App\\Entity\\ProcurationRequest::DISABLED_REASON_BY_PROCURATION_MANAGER') ? 'Manuellement' : 'Automatiquement' }}
+            </td>
+        {% endif %}
```
[the remark this answers is not in the document]
Comment (illusionOfParadise) [reply templates/procuration_manager/_requests_list.html.twig:41]: There's no point to do this, the list diplays either enabled requests or disables ones, to add an empty `<td></td>` is take up space for noting.

File: templates/procuration_manager/requests.html.twig [hunk 4]
```diff
@@ -28,6 +28,11 @@
                 {{ '{1} Vous avez une demande de procuration à traiter.|[2, Inf[ Vous avez %count% demandes de procuration à traiter.'|transchoice(total_count) }}
             {% endif %}
         </div>
+    {% else %}
+        <div class="alert alert--tips b__nudge--bottom-60">
+            Vous avez la possibilité de réactiver un mandant à tout instant.<br />
+            Néanmoins certains mandants ont été désactivés de manière automatique pour des raisons de sécurité, merci de ne pas les réactiver.
```
[the remark this answers is not in the document]
Comment (illusionOfParadise) [reply templates/procuration_manager/requests.html.twig:34]: The text has been provided by PO.

**Review by ottaviano — APPROVED** [review]

File: templates/procuration_manager/requests.html.twig (hunk 4 above)
[the remark this answers is not in the document]
Comment (elvino29) [reply templates/procuration_manager/requests.html.twig:34]: ok

**Pull request merged**